e2e/framework-pandas/client.py

```diff
@@ -5,7 +5,7 @@
 
 import flwr as fl
 
-df = pd.read_csv("./data/client.csv")
```

Comment:
I think this path should be relative as the test otherwise probably breaks locally.

Reply:
It does not work with relative path for now. I talked with @jafermarq about this, for now this is the way to make it pass.

Reply:
Dataset is moved to tmp.